massgov/openmass
# DP-22007: SPIKE Perform assessment of entity usage module

**Description:**
Discover work in progress.


**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-22007


**To Test:**
- Login and go to /doc/dcr-media-advisory-dcr-officials-welcome-returning-massachusetts-firefighters/mass-usage.
- Verify the tab is “Pages linking here”.
- Verify the table headers are:
- Entity
- ID
- Content Type
- Field name
- Status
- Verify the data looks accurate for each table heading.
- Verify there is one result (this should match the Descendant Manager “Pages linking here” on Production).
- Go to /orgs/massachusetts-department-of-revenue/mass-usage.
- Verify the tab is “Pages linking here”.
- Verify the table headers are:
- Entity
- ID
- Content Type
- Field name
- Status
- Verify the data looks accurate for each table heading.
- Verify the results match the Descendant Manager “Pages linking here” on Production.
- Verify the results are in alphabetical, ascending order by the Entity column.
- Verify the pager works as expected.

**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
